eox_tenant/test/tutor/integration.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Install the package in the edxapp env
```

Comment:
This is merely a suggestion, but I'm not sure if the inline comments on lines 3, 7, and 11 add much value. The script is easy to understand, so perhaps a single inline comment at the beginning briefly explaining what the script does would be enough.

Reply:
thank you! corrected in all PR's